src/utils/isMobile.ts

```diff
@@ -20,15 +20,12 @@ function match(regex, userAgent) {
 }
 
 function isMobile(userAgent?: string) {
-  
-  if (typeof window === "undefined" || typeof navigator === "undefined") return userAgent || false;
-  
```

Comment:
Please do not remove this, it will report error in SSR.

Reply:
I copy this code from another library. should be fine, but returns wrong data type for this function `isMobile` returns an object instead of boolean.